web/src/cypress/integration/admin.ts

```diff
@@ -1,12 +1,67 @@
 import { Chance } from 'chance'
-const c = new Chance()
 import { testScreen } from '../support'
+const c = new Chance()
 
 testScreen('Admin', testAdmin, false, true)
 
 function testAdmin(screen: ScreenFormat) {
-  let cfg: Config
-  describe('Admin Page', () => {
+  describe('Admin System Limits Page', () => {
+    let limits: Limits[]
+    beforeEach(() => {
+      cy.getLimits().then(l => {
+        limits = l
+        return cy.visit('/admin/limits')
+      })
+      return cy.visit('/admin/limits')
+    })
+
+    it('should allow updating system limits values', () => {
+      const newContactMethods = c.integer({ min: 0, max: 1000 })
+      const newEPActions = c.integer({ min: 0, max: 1000 })
+
+      cy.form({
+        ContactMethodsPerUser: newContactMethods.toString(),
+        EPActionsPerStep: newEPActions.toString(),
+      })
+      cy.get('button[data-cy=save]').click()
```

Comment:
```suggestion
      cy.dialogClick('Save')
```

Reply:
These buttons are in the top bar, not in a dialog.